core/src/test/java/org/apache/iceberg/avro/TestAvroDeleteWriters.java

```diff
@@ -56,9 +56,9 @@ public class TestAvroDeleteWriters {
 
   private List<Record> records;
 
-  @Rule public TemporaryFolder temp = new TemporaryFolder();
+  @TempDir java.nio.file.Path temp;
```

Comment:
do we need the fully qualified name here? I think we can just add an import here and in the other files